src/Grid/Grid.js

```diff
@@ -69,6 +91,14 @@ class Grid extends React.Component {
                 </TableRow>
               ))}
           </TableBody>
+          { 
+            showFooter === true &&
+              <TableFooter>
+                <TableRow>
+                  <TablePagination count = { totalCount } rowsPerPage = { rowsPerPage } page = { page } onChangePage = { this.handleChangePage } onChangeRowsPerPage = { this.handleChangeRowsPerPage } />
```

Comment:
As I already stated at https://github.com/unosquare/tubular-react/issues/19, **Pagination should not be part of the table footer**.

It would be good to have something like

<Grid dataSource="dataSource" columns="columns">
    <TableFooter>
            <TableRow>
              <TableRowColumn>ID</TableRowColumn>
              <TableRowColumn>Name</TableRowColumn>
              <TableRowColumn>Status</TableRowColumn>
            </TableRow>
            <TableRow>
              <TableRowColumn>{aggregatedValues.IdCount}</TableRowColumn>
              <TableRowColumn>{aggregatedValues.Dummy1}</TableRowColumn>
              <TableRowColumn>{aggregatedValues.Dummy2}</TableRowColumn>
            </TableRow>
       </TableFooter>
</Grid>

Please take a look at https://unosquare.github.io/tubular/#!/Basic => Grid with Aggregation results to see how Tubular is retrieving Aggregated values.

Reply:
Totally agree... I also said before, the footer is an independent component.